Maybe uninitialized error when compiling in debug mode with GCC 4.6.0

This error occurs only when invoking make with config set to debug. I am using `gcc (Solus) 6.4.0` and `LLVM 3.9.1`.
```
genoperator.c
src/libponyc/codegen/genoperator.c: In function ‘make_divmod’:
src/libponyc/codegen/genoperator.c:188:5: error: ‘post_block’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
     LLVMPositionBuilderAtEnd(c->builder, post_block);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```

--- a/src/libponyc/codegen/genoperator.c
+++ b/src/libponyc/codegen/genoperator.c
@@ -133,7 +133,7 @@ static LLVMValueRef make_divmod(compile_t* c, ast_t* left, ast_t* right,
   LLVMBasicBlockRef insert;
   LLVMBasicBlockRef nonzero_block;
   LLVMBasicBlockRef no_overflow_block;
-  LLVMBasicBlockRef post_block;
+  LLVMBasicBlockRef post_block = NULL;
   LLVMValueRef zero;
 
   if(safe)

